src/script.ts

```diff
@@ -68,6 +75,8 @@ export function isLoaded() {
 
 // load the ArcGIS API on the page
 export function loadScript(options: ILoadScriptOptions = {}): Promise<HTMLScriptElement> {
+  options = { ...defaultOptions, ...options };
```

Comment:
Let's not mutate the options that are passed in. Instead, let's use `const opts = { ...defaultOptions, ...options };` here and then use `opts` throughout the rest of this fn.

Reply:
Good call 👍